lib/internal/encoding.js

```diff
@@ -419,10 +421,13 @@ function makeTextDecoderICU() {
       this[kFatal] = Boolean(options?.fatal);
       // Only support fast path for UTF-8.
       this[kUTF8FastPath] = enc === 'utf-8';
+      this[kLatin1FastPath] = enc === 'windows-1252';
       this[kHandle] = undefined;
 
-      if (!this[kUTF8FastPath]) {
-        this.#prepareConverter();
```

Comment:
This change is wrong. prepareConverter should only be called if we don't have fast path.

Reply:
thank you